pom.xml

```diff
@@ -270,6 +273,12 @@
         <groupId>org.glassfish.jersey.media</groupId>
         <artifactId>jersey-media-json-processing</artifactId>
         <version>${org.glassfish.jersey.version}</version>
+        <exclusions>
```

Comment:
I'm not a fan of doing this. What is the parsson version coming with jersey 3.1.5 and what is the added value of overriding it?

Reply:
hi @jendib the parsson library is a transitive dependency inside the jersey. The parsson library had a vulnerability which only gets solved by upgrading it. So i excluded the transitive dependency and added the upgraded dependency.